package.json

```diff
@@ -79,8 +79,8 @@
     "@types/node-emoji": "^1.8.1",
     "@types/sortablejs": "^1.15.0",
     "@types/uuid": "^8.3.4",
-    "@typescript-eslint/eslint-plugin": "5.57.1",
-    "@typescript-eslint/parser": "5.57.1",
+    "@typescript-eslint/eslint-plugin": "5.58.0",
+    "@typescript-eslint/parser": "5.58.0",
```

Comment:
question: did you mean to bump the versions here? Unrelated to the changes in this PR

Reply:
How did you update? I wouldn't think it would show this change if it's up to date with main 🤔